### DISCUSSION TITLE
Lock prevents run_cli in 0.9.0

### DISCUSSION THREAD

**Discussion by mc3** [AsamK/signal-cli#962] [Q&A] [answered]
Starting the daemon via rc script and then issuing any run_cli gets this message and waits for ever:
```root@ping:~ # service signal_cli onestart
+<own phone #>
/usr/sbin/daemon -p /var/run/signal_cli.pid -u signal-cli /usr/local/bin/signal-cli --config /var/lib/signal-cli -u +<own phone #> daemon --system
Starting signal_cli.
root@ping:~ # INFO DaemonCommand - Exported dbus object: /org/asamk/Signal

root@ping:~ # service signal_cli oneruncli send -m 'TEST via daemon' +<target phone #>
INFO SignalAccount - Config file is in use by another instance, waiting…
```
This is signal-cli-0.9.0_1,1 and libsignal-client-0.9.6_6,1 from FreeBSD ports running on FreeBSD 13.0-RELEASE-p11
Verbose log attached
[report.txt](https://github.com/AsamK/signal-cli/files/8790829/report.txt)
.

**Comment by AsamK** [comment, marked as answer]
Only one signal-cli instance can be running for one account at the same time.
When signal-cli is running in daemon mode, you can can interact with it via dbus.
Either use dbus language bindings to do that or use signal-cli with the `--dbus`/`--dbus-system` flag.
for your setup this should work: `service signal_cli oneruncli --dbus-system send -m 'TEST via daemon' +<target phone #>`

**Reply by mc3** [reply]
Yes it works perfectly!
With the --system option in the rc script, I do not need the --dbus.system option in runcli anymore.

Thanks for your work!
